app/views/payments/_payment.html.haml

```diff
@@ -0,0 +1,35 @@
+.div
+  .col-md-12.table-responsive
+    %table.table.table-hover
+      %thead
+        %tr
+          %th Ticket
+          %th Quantity
+          %th Price
+          %th Total
+      %tbody
+      - @unpaid_ticket_purchases.each do |ticket|
+        %tr
+          %td
+            = ticket.title
+          %td
+            = ticket.quantity
+          %td
+            = humanized_money_with_symbol ticket.price
+          %td
+            = humanized_money_with_symbol ticket.quantity * ticket.price
+
+= form_tag conference_payments_path do
+  %ul.price-tags.pull-right
+    %li
+      %a
+        = humanized_money_with_symbol @total_amount_to_pay
+  %script.stripe-button{ src: "https://checkout.stripe.com/checkout.js",
+        data: { amount: @total_amount_to_pay.cents,
+          currency: @total_amount_to_pay.currency,
+          image: image_url('OSEM_ICON.jpg'),
```

Comment:
The icon is really cute, but please take into consideration that OSEM is not exclusively used by openSUSE. We should not force openSUSE logo here. The name of the file also seems to say OSEM_icon, but it is really openSUSE icon. We could use OSEM logo instead.